pkg/cloudprovider/vsphere/cloud.go

```diff
@@ -17,18 +17,20 @@ limitations under the License.
 package vsphere
 
 import (
-	"fmt"
 	"io"
+	"io/ioutil"
 	"os"
 	"runtime"
 
 	v1 "k8s.io/api/core/v1"
-	cloudprovider "k8s.io/cloud-provider"
 	"k8s.io/klog"
 
-	"github.com/vmware/vsphere-automation-sdk-go/runtime/log"
+	cloudprovider "k8s.io/cloud-provider"
 
+	"github.com/vmware/vsphere-automation-sdk-go/runtime/log"
```

Comment:
non-blocking nit: we should group github.com imports separately.

Reply:
Fixed